src/Claims.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @public
+ */
+export type Mandatory<T, K extends keyof T> = Required<Pick<T, K>> & Omit<T, K>;
```

Comment:
The `export * from './Claims'` from the library entrypoint causes this to be exported as part of the public API. GIven that `UserProfile` shouldn't need this type with the above comment, I suggest keeping this type local to this file.

Reply:
`UserProfile` needs to be exported, and as it extends from `IdTokenClaims` so does `IdTokenClaims`, `Madatory`, `OidcStandardClaims` and `JwtClaims`.

Otherwise the following would appear in the APIs doc:
```md
// Warning: (ae-forgotten-export) The symbol "Mandatory" needs to be exported by the entry point index.d.ts
// Warning: (ae-forgotten-export) The symbol "OidcStandardClaims" needs to be exported by the entry point index.d.ts
// Warning: (ae-forgotten-export) The symbol "JwtClaims" needs to be exported by the entry point index.d.ts
```

Reply:
`export * from './Claims'` is fine, I just don't want a utility type like `Mandatory` being exported by this library. In this case the` (ae-forgotten-export) The symbol "Mandatory" needs to be exported by the entry point index.d.ts` is acceptable